airbyte-db/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -169,13 +169,22 @@ public static void setStatementField(PreparedStatement preparedStatement,
     switch (cursorFieldType) {
       // parse date, time, and timestamp the same way. this seems to not cause an problems and allows us
```

Comment:
nit: update this comment

Reply:
can we also leave a comment here summarising what we discussed yesterday around potential errors using Date as a cursor?